pkg/driver/executor/executor.go

```diff
@@ -207,26 +205,27 @@ func (ex *Executor) deployServer(ctx context.Context, machineName string, userDa
 	flavorName := ex.Config.Spec.FlavorName
 
 	var (
-		imageRef   string
-		createOpts servers.CreateOptsBuilder
-		err        error
+		imageRef string
+		err      error
+		hintOpts volumes.SchedulerHintOpts
```

Comment:
I think that the use of the volumes package here for scheduler hints is not correct.